transformer/transformers/ampruntimejs.go

```diff
@@ -0,0 +1,37 @@
+package transformers
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/ampproject/amppackager/transformer/internal/amphtml"
+	"github.com/ampproject/amppackager/transformer/internal/htmlnode"
+	"golang.org/x/net/html"
+	"golang.org/x/net/html/atom"
+)
+
+// AMPRuntimeJS rewrites the value of src in script nodes, where applicable.
+// If the value is of the form "*.js", replace it with "*.sxg.js".
+func AMPRuntimeJS(e *Context) error {
+	for n := e.DOM.RootNode; n != nil; n = htmlnode.Next(n) {

```

Comment:
We should be able to traverse the children on of the `<head>` directly, allowing us to skip the entire body.

Reply:
Would that just be `e.DOM.Head`?

Reply:
I found it, it's `e.DOM.HeadNode`.